packages/common-starknet/src/codegen/index.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright 2020-2024 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+// export * from './codegen-controller';
```

Comment:
It should be possible to use typechain here too https://github.com/dethcrypto/TypeChain/tree/master/packages/target-starknet